SKILL.tsv

```diff
@@ -4058,8 +4058,8 @@ SKILL_20150428_004057	{memo X}$Evasion: -#{CaptionRatio}#{nl}Pointing AI Duratio
 SKILL_20150428_004058	$Detects hidden monsters {nl}Hounding AI Duration: 90 seconds
 SKILL_20150428_004059	{memo X}$Consumes #{SpendPoison}# Poison Pot Poison
 SKILL_20150428_004060	{memo X}Attack #{SkillAtkAdd}#{nl}Duration 30 seconds
-SKILL_20150428_004061	Throw tear gas on target area. Target falls under Darkness.
-SKILL_20150428_004062	$Tear Gas Duration: #{CaptionTime}# seconds
+SKILL_20150428_004061	Throw Lachrymator on target area. Target falls under Darkness.
```

Comment:
This can be simplified since this is a description of the Lachrymator skill.  

I suggest changing it to: "Throw tear gas at a targeted area. The target becomes afflicted with Darkness."